### DISCUSSION TITLE
Sponsorship, what are the asks?

### DISCUSSION THREAD

**Discussion by plaughton** [cdmoro/bootstrap-vue-3#1406] [General]
We are in the process of moving our Nuxt 2/Bootstrap Vue app to Nuxt 3/Bootstrap Vue Next.

Our biggest issues are 
1. Sort on tables (but not with that hard-to-change icon on bootstrap-vue)
2. Implementation of BPopover directives (these do not seem to work for us)

But still BVN is great, and we think will be mature enough when we hope to release in October. 

We would like to support, esp given that we are likely to use in production. Would be great to see how this can help the active contributors (e.g. VividLemon). Are there any specific asks other than a regular contribution?

**Comment by xvaara** [comment]
How BPopover directives aren't working for you?

**Reply by VividLemon** [reply]
Could you write the docs for both tooltip and popover

**Reply by xvaara** [reply]
Yes, when I have time 😄
